apps/crn-server/src/types.ts

```diff
@@ -0,0 +1,5 @@
+export type ResearchOutputFilter = {
```

Comment:
this type has no reason to sit in a separate file and should be part of `apps/crn-server/src/utils/odata.ts`